### PR TITLE
[Automated Integration]Merge Azure module changes from Ansible repo

### PR DESCRIPTION
**Pull request by yungezz**
@yungezz @yuwzho @zikalino

### CODE REVIEW

**Comment by yungezz** [comment]
@yuwzho

File: .github/sha.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "lib/ansible/module_utils/azure_rm_common.py": "1111111111111111111111111111111111111111", 
-  "lib/ansible/modules/cloud/azure": "1111111111111111111111111111111111111111", 
-  "packaging/requirements/requirements-azure.txt": "1111111111111111111111111111111111111111", 
-  "test/integration/targets": "1111111111111111111111111111111111111111"
+  "lib/ansible/module_utils/azure_rm_common.py": "c29fe66edf7972dc8dd49f6216af2a2cfbe6add4", 
+  "lib/ansible/modules/cloud/azure": "133b70ff4503be52b007d3abe642d7d7f360b686",
```
Comment (zikalino) [inline .github/sha.json]: should this file be actually checked in?

File: .travis.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ env:
     - message=()
     - ANSIBLE_VERSION=latest
     - max_retry_count=2
-    #- ANSIBLE_VERSION=2.4.2.0
+    - ANSIBLE_VERSION=2.5.5
```
Comment (zikalino) [inline .travis.yml]: why not 2.6?

File: .github/sha.json (hunk 1 above)
Comment (yungezz) [reply .github/sha.json]: yes, it's used to save where's last commit synced

File: .travis.yml (hunk 2 above)
Comment (yungezz) [reply .travis.yml]: updated, it's used for testing

**Pull request merged**